main.py

```diff
@@ -2,11 +2,16 @@
 detect faces and draw a box around them
 """
 import sys
+import time
+import logging
 import pickle
 import cv2
 import face_recognition
 import functions
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
a comment may be needed about how to get into debug mode

Reply:
How about this?

```
# Logging levels affect many dependencies as well as current script.
# Change the verbosity of logs by changing the level to one of the following:
# CRITICAL, ERROR, WARNING, INFO, DEBUG, NOTSET
# Whatever level is set, that level and all levels more severe it will be logged.
```